controller/addons/custom-facebook-feed/custom-facebook-feed.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace cookiebot_addons_framework\controller\addons\custom_facebook_feed;
+
+class Custom_Facebook_Feed {
+
+	public function __construct() {
+		/**
+		 * We add the action after wp_loaded and replace the original Custom
+		 * Facebook Feed action with our own adjusted version.
+		 */
+		add_action( 'wp_loaded', array( $this, 'cookiebot_addon_custom_facebook_feed' ), 5 );
+	}
+
+	/**
+	 * Manipulate the scripts if they are loaded.
+	 *
+	 * @since 1.1.0
+	 */
+	public function cookiebot_addon_custom_facebook_feed() {
+		//Check if Custom Facebook Feed is loaded.
+		if ( ! shortcode_exists( 'custom-facebook-feed' ) ) {
+			return;
+		}
+		//Check if Cookiebot is activated and active.
+		if ( ! function_exists( 'cookiebot_active' ) || ! cookiebot_active() ) {
+			return;
+		}
+
+		//Remove cff_js action and replace it with our own
+		if ( has_action( 'wp_footer', 'cff_js' ) ) {
+			cookiebot_buffer_output( 'wp_footer', 99 );
```

Comment:
**cookiebot_buffer_output( 'wp_footer', 10, array( 'cfflinkhashtags' ) );**

Few things:
- The priority needs to be the same as the real action hook. So you need to set it to 10. 
FYI Basically we will automatically create 2 new hooks to manipulate the output which will be created in wp_footer(priority 10). These hooks will have priorities 9(start reading) and 11(stop and return manipulated data).
- As third parameter, you can give keywords which are only used in that javascript file. (for example: cfflinkhashtags). That way we can detect the script we need to overwrite with custom attributes.